### PR TITLE
Login Modal, Single wishcard #70 - WIP 

### PR DESCRIPTION
**Pull request by stacysealky**
WIP but you can still review and merge.
Going to make more design changes, add a toggle bar, and turn the signup page into a modal as well.

### CODE REVIEW

File: src/components/authentication/FormInput.jsx [hunk 1]
```diff
@@ -2,10 +2,16 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 export default function FormInput(props) {
-  const { containerClass, type, name, registerFunc, placeholder, errorMsg } = props;
+  const { containerClass, type, name, registerFunc, placeholder, errorMsg, icon, inputClass } = props;
   return (
-    <div className={`${containerClass} my-5 px-3 form-group`}>
-      <input type={type} className="form-control form-control-lg" {...registerFunc(name)} placeholder={placeholder} />
+    <div className={`${containerClass} my-4 px-3 form-group`}>
+      {icon}
```
Comment (Enubia) [inline src/components/authentication/FormInput.jsx:8]: please make this conditional as we don't always have icons on a input field

File: src/styles/pages/_login.scss [hunk 2]
```diff
@@ -32,3 +33,37 @@
...
+.login {
+  .row {
+    margin-bottom: 3em;
+  }
+  .col-8 {
+    border-right: solid 2px $gray-500;
+    margin: 2em 0;
+  }
+  h1 {
+    color: $primary !important;
```
Comment (Enubia) [inline src/styles/pages/_login.scss:46]: this can be done with pure bootstrap classes e.g. ```text-primary text-center mb-2```

**Review by Enubia — CHANGES_REQUESTED** [review]

**Comment by Enubia** [comment]
I would keep the signup page still in case users get redirected to us from any other site, i did the same with the login page

File: src/components/authentication/FormInput.jsx (hunk 1 above)
Comment (stacysealky) [reply src/components/authentication/FormInput.jsx:8]: It is optional

File: src/styles/pages/_login.scss (hunk 2 above)
Comment (stacysealky) [reply src/styles/pages/_login.scss]: changed

**Comment by stacysealky** [comment]
> I would keep the signup page still in case users get redirected to us from any other site, i did the same with the login page

Yes, that makes sense. Let me think about this flow today.

**Pull request merged**